EIPS/eip-7212.md

```diff
@@ -169,7 +169,7 @@ geomean                                               33.00        7.000
 
 ## Reference Implementation
 
-Implementation of the `P256VERIFY` precompiled contract is applied to go-ethereum client to create a reference. Also, an “secp256r1” package has already been included in the Besu Native library which is used by Besu client. Other client implementations are in the future roadmap.
+Implementation of the `P256VERIFY` precompiled contract is applied to go-ethereum client to create a [reference](TODOLINK). Also, an “secp256r1” package [has already been included in the Besu Native library](TODOLINK) which is used by Besu client. Other client implementations are in the future roadmap.
```

Comment:
could someone share the source for these?

Reply:
It's the [reference implementation](https://github.com/ethereum/go-ethereum/pull/27540), but external links are not allowed in the EIP itself. We can add as just text, not hyperlink.

Reply:
gotcha, thanks, removing this then.